src/components/CommentsItem.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <div class="comments-item">
```

Comment:
play with styles - no border bottom, no strange icon. Some user info (eg name, time, upvotes - inspire from reddit and ionic2-reddit-reader)

Collapse/expand would be good